examples/energy-management-app/energy-management-common/include/EnergyEvseDelegateImpl.h

```diff
@@ -236,9 +309,9 @@ class EnergyEvseDelegate : public EnergyEvse::Delegate
 
 private:
     /* Constants */
-    static constexpr int kDefaultMinChargeCurrent                     = 6000;                  /* 6A */
-    static constexpr int kDefaultUserMaximumChargeCurrent             = kMaximumChargeCurrent; /* 80A */
-    static constexpr int kDefaultRandomizationDelayWindow             = 600;                   /* 600s */
+    static constexpr int kDefaultMinChargeCurrent                     = 6000;  /* 6A */
+    static constexpr int kDefaultUserMaximumChargeCurrent             = 80000; /* 80A */
```

Comment:
constants should have unit of measurement in name. these seem to be `_mA` or `_milliAmp` or something like that.

`kDefaultRandmizationDelayWindow` should have a `Sec` or `Seconds` suffix.

Reply:
Change made.